docs/deploying-airbyte/deploying-airbyte.md

```diff
@@ -0,0 +1,182 @@
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Deploying Airbyte
+
+The Airbyte platform is a sophisticated data integration platform that enables you to handle large amounts of data movement. If setting up an Airbyte server does not fit your usecase needs (i.e. you're using Jupyter Notebooks or iterating on an early prototype for your project) you may find the [PyAirbyte](./using-airbyte/pyairbyte/getting-started) documentation useful. To quickly deploy Airbyte on your local machine you can visit the 
+[Quickstart](./using-airbyte/getting-started/oss-quickstart) guide.
+
+:::tip
+Enterprise Customers should follow the steps outlined in our docs on [Airbyte Self-Managed Enterprise](../enterprise-setup/README.md) and the associated [implementation guide](../enterprise-setup/implementation-guide.md).
+:::
+
+## Understanding the Airbyte Deployment
+
+Airbyte is built to be deployed on top of Kubernetes in a cloud environment. We recommend deploying Airbyte using Helm and the documented Helm chart values. 
+
+Helm is a Kubernetes package manager for automating deployment and management of complex applications with microservices on Kubernetes.  Refer to our [Helm Chart Usage Guide](https://airbytehq.github.io/helm-charts/) for more information about how to get started.
+
+
+The [Infrastructure](#deploying-airbyte/infrastructure) section describes the Airbyte's recommended cloud infrastructure to set up for each supported platform. Keep in mind that these guides are meant to assist you, but you are not required to follow them. Airbyte is designed to be as flexible as possible in order  to fit into your existing infrastructure.
+
+## Integrations
+
+The Airbyte platform is built to integrate with your existing cloud infrastructure. You can 
+configure various components of the platform to suit your needs. This includes an object store,
+such as S3 or GCS for storing logs and state, a database for externalizing state, and a secret 
+manager for keep your secrets secure. Each of these integrations can be configured to suit your 
+needs. Their configuration is described in the [Integrations](#deploying-airbyte/integrations) 
+section. Each of these integrations has its own section where you'll find an explanation of the rationale for why it's useful to configure the integration. There, you'll also find details about how to configure the integration.
+
+<details>
+<summary> Integration Section Links </summary>
+
+- [State and Logging Storage](./deploying-airbyte/integrations/storage)
```

Comment:
Looks like this link is broken - could you update?

Reply:
I think it might be because you're already in ./deploying-airbyte? The preview seems to be trying to go to deploying-airbyte/deploying-airbyte/integrations/..